src/components/Charts.js

```diff
@@ -1,10 +1,9 @@
 import React, { Component } from "react";
-import { Row, Col, Container } from "reactstrap";
+import { Row, Col } from "reactstrap";
 import Piechart from "./Piechart";
 import Barchart from "./Barchart";
 
 class Charts extends Component {
-  state = {};
   render() {
     return (
```

Comment:
There is no reason to keep it as a class component. If you don't need state, you can convert it to a dumb component too.

Reply:
honestly, i haven't decided yet that either we make this component stateful or stateless. but for now, let me make it stateless component